test/appengine/test_gae_connectionpool.py

```diff
@@ -0,0 +1,193 @@
+import unittest
+import logging
+import sys
+
+from nose.plugins.skip import SkipTest
+
+try:
+    from google.appengine.api import urlfetch
+except ImportError:
+    raise SkipTest("App Engine SDK not available.")
```

Comment:
Isn't this file only executed if we explictly want to test GAE?
If so we should blow up.